colpali_engine/trainer/colmodel_training.py

```diff
@@ -10,8 +10,8 @@
 from tqdm import tqdm
 from transformers import AutoTokenizer, Idefics2Processor, PreTrainedModel, PreTrainedTokenizer, TrainingArguments
 
-from colpali_engine.collators.custom_collator import CustomCollator
 from colpali_engine.collators.hard_neg_collator import HardNegCollator
+from colpali_engine.collators.visual_retriever_collator import CustomCollator
```

Comment:
Might as well rename the collator to something more explicit

Reply:
Any proposition instead of `VisualRetrieverCollator`?

Reply:
Nah nice